GoMeet/src/main/webapp/js/login-status.js

```diff
@@ -2,8 +2,17 @@ class LoginStatus {
   
   static endpoint = '/user-status'; 
 
-  static async doGet() {
-    let promise = await fetch(LoginStatus.endpoint).then(response => response.json());
-    return promise; 
+  static async doGet(meetingEventId) {
+    if (meetingEventId === null || meetingEventId === undefined) {
+      throw new Error(INSUFFICIENT_REQUEST_PARAM); 
+    }
+
+    if (typeof meetingEventId !== 'string') {
+      throw new Error(INVALID_PARAM_TYPE); 
+    }
+
+    let urlString = DAOUtils.url(LoginStatus.endpoint, {'meetingEventId': meetingEventId});
+    let results = await fetch(urlString).then((results) => results.json());
```

Comment:
Have you handled the case where this call to fetch() fails?

Reply:
Added a check in the servlet for bad fetch requests